src/appium.sh

```diff
@@ -372,5 +372,5 @@ elif [ "$GENYMOTION" = true ]; then
 	esac
 else
 	echo "Using Emulator"
-	python3 -m src.app
+	bash /root/src/run_app.sh
```

Comment:
I think this one can be deleted as the whole installation will go to Dockerfile and pip install with go to requirements.txt

Reply:
Done,  I removed this logic since `toml` package should be now installed from `requirements.txt`